### ISSUE TITLE
Support for sentry 7.x

### ISSUE THREAD

**Issue by aadis** [bogdal/sentry-youtrack#8]
Hi,

We at ideadevice love sentry-youtrack to help us make sure we track & fix all unhandled errors. We recently upgraded to sentry 7.1 and the extension has been broken since. 

Any chance of getting it upgraded?

**Comment by bogdal** [comment]
Hi @aadis. I'm glad you're happy with this plugin. Thanks for info. I will look into it.

Cheers!

**Comment by bogdal** [comment]
@aadis new version is ready and available on [pypi](https://pypi.python.org/pypi/sentry-youtrack/0.2.3). Check it out.

**Comment by aadis** [comment]
Hiya,
Thanks a ton! Will try out by tomorrow and let you know. 
Regards
--
Aaditya Sood
CEO, Idea Device
US: +1 (650) 784-0420
India: +91 9845100258
www.ideadevice.com | www.calm.io





On Mon, Feb 9, 2015 at 3:15 PM -0800, "Adam Bogdał" <notifications@github.com> wrote:












@aadis new version is ready and available on pypi. Check it out.

**Comment by aadis** [comment]
hi,

I'm now getting:
```
[ERROR] Internal Server Error: /idea-device/democalmio/plugins/youtrack/
Traceback (most recent call last):
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/sentry/web/decorators.py", line 182, in _wrapped
    return func(request, *args, **kwargs)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/sentry/web/frontend/projects/plugins.py", line 72, in configure_project_plugin
    action, view = plugin_config(plugin, project, request)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/sentry/web/helpers.py", line 203, in plugin_config
    }, context_instance=RequestContext(request))))
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/template/loader.py", line 169, in render_to_string
    return t.render(context_instance)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/template/base.py", line 134, in _render
    return self.nodelist.render(context)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/crispy_forms/templatetags/crispy_forms_tags.py", line 207, in render
    c = self.get_render(context)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/crispy_forms/templatetags/crispy_forms_tags.py", line 119, in get_render
    response_dict = self.get_response_dict(helper, context, is_formset)
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/crispy_forms/templatetags/crispy_forms_tags.py", line 160, in get_response_dict
    'template_pack': settings.CRISPY_TEMPLATE_PACK,
  File "/home/ops/.virtualenvs/sentry7/local/lib/python2.7/site-packages/django/conf/__init__.py", line 55, in __getattr__
    return getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'CRISPY_TEMPLATE_PACK'
```

**Comment by aadis** [comment]
My pip freeze output:
```
BeautifulSoup==3.2.1
Django==1.6.10
Markdown==2.4.1
Pygments==1.6
South==1.0.1
amqp==1.0.13
anyjson==0.3.3
argparse==1.2.1
billiard==2.7.3.34
celery==3.0.25
cffi==0.9.0
cryptography==0.7.2
cssselect==0.9.1
cssutils==0.9.10
django-bitfield==1.7.1
django-celery==3.0.23
django-crispy-forms==1.4.0
django-jsonfield==0.9.13
django-paging==0.2.5
django-picklefield==0.3.1
django-recaptcha==1.0.3
django-social-auth==0.7.28
django-statsd-mozilla==0.3.8.7
django-sudo==1.1.3
django-templatetag-sugar==1.0
djangorestframework==2.3.14
email-reply-parser==0.2.0
enum34==0.9.23
exam==0.10.5
gevent==1.0.1
greenlet==0.4.5
gunicorn==19.3.0
httplib2==0.9
ipaddr==2.1.11
kombu==2.5.16
logan==0.5.10
lxml==3.4.2
mock==1.0.1
ndg-httpsclient==0.3.3
nydus==0.10.8
oauth2==1.5.211
progressbar==2.3
psycopg2==2.5.4
py==1.4.26
pyOpenSSL==0.14
pyasn1==0.1.7
pycparser==2.10
pytest==2.6.4
pytest-django==2.8.0
python-dateutil==1.5
python-memcached==1.53
python-openid==2.2.5
pytz==2014.10
raven==5.2.0
redis==2.10.3
requests==2.5.3
sentry==7.4.0
sentry-searchbutton==0.1.0
sentry-webhooks==0.3.1
sentry-youtrack==0.2.3
setproctitle==1.1.8
simplejson==3.3.3
six==1.9.0
statsd==1.0.0
toronado==0.0.6
uWSGI==2.0.9
ua-parser==0.3.6
urllib3==1.7.1
wsgiref==0.1.2
```

**Comment by bogdal** [comment]
Hi @aadis. This is a bug of `django-crispy-forms` package (`1.4.0`), see https://github.com/maraujop/django-crispy-forms/issues/237. This was fixed in version `1.5.0` which is not available on pypi yet. Sentry since `7.2.0` uses the buggy version of this package.

The easiest way to solve this is add `CRISPY_TEMPLATE_PACK = 'bootstrap'` to the `sentry.conf.py` file or.. downgrade sentry to the `7.1.4` version :)

Cheers!
